wolfssl/wolfcrypt/blake2-impl.h

```diff
@@ -86,7 +86,7 @@ static WC_INLINE void store32( void *dst, word32 w )
 static WC_INLINE void store64( void *dst, word64 w )
 {
 #if defined(LITTLE_ENDIAN_ORDER)
-  *( word64 * )( dst ) = w;
+  XMEMCPY(dst, &w, sizeof(word64));
```

Comment:
`memcpy` wrap in `XSTREAM_ALIGN`?

Reply:
Replace with aligned assignments like in sha3.c:Load64BitBigEndian().
Obviously this is a little endian assignment so is a little different.

Reply:
Don't use XMEMCPY.
Check if alignment is large enough for this path or whether the byte-by-byte is to be used.
Could also have multiple alignment paths like SHA-3 code.

Reply:
To have just a single implementation for those operations would it make sense to live in `misc.c`?

Reply:
Yes. We have code for big-endian operations in misc.c already (not optimized).